api/tests/routes/pro/get_offerer_headline_offer_test.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offers import factories as offers_factories
+from pcapi.core.testing import assert_num_queries
+from pcapi.core.users import factories as users_factories
+
+
+class Return200Test:
+    num_queries = 8
+
+    @pytest.mark.usefixtures("db_session")
+    def test_get_offerer_headline_offer_success(self, client):
+        user_offerer = offerers_factories.UserOffererFactory()
+        pro = user_offerer.user
+        offerer = user_offerer.offerer
+        venue = offerers_factories.VenueFactory(managingOfferer=offerer)
+        offer = offers_factories.OfferFactory(venue=venue)
+        offers_factories.HeadlineOfferFactory(offer=offer, venue=venue)
+        client = client.with_session_auth(email=pro.email)
+        with assert_num_queries(self.num_queries):
+            response = client.get(f"/offerers/{offerer.id}/headline-offer")
```

Comment:
Tu peux enlever 1 query qu'on ne veut pas tester ici en faisant 
```suggestion
        offerer_id = offerer.id
        with assert_num_queries(self.num_queries):
            response = client.get(f"/offerers/{offerer_id}/headline-offer")
```